src/components/elements/nav/Menu.tsx

```diff
@@ -1,36 +1,97 @@
-import {Button, makeStyles} from '@material-ui/core';
 import React from 'react';
+
+import {makeStyles} from '@material-ui/core';
+import {useSelector, useDispatch} from 'react-redux';
 import {Link as RouterLink} from 'react-router-dom';
-import {navItems} from './Items';
+
+import {authDispatchers} from '../../../state/auth/dispatchers';
+import {rootState} from '../../../state/reducer';
+import UIButton from '../../elements/ui/Button';
+import {navBarItems} from './Items';
 
 const useStyles = makeStyles(() => ({
   menuButton: {
+    width: '80px',
+    fontFamily: 'Open Sans, sans-serif',
+    fontWeight: 700,
+    size: '18px',
+    marginLeft: '38px',
+  },
+  menuFlex: {
+    display: 'flex',
+    flexDirection: 'row',
+  },
+  signOutButton: {
+    width: '150px',
     fontFamily: 'Open Sans, sans-serif',
     fontWeight: 700,
     size: '18px',
     marginLeft: '38px',
   },
 }));
 
+const IsNotLoggedInItems = () => {
+  const style = useStyles();
+  return (
+    <>
```

Comment:
This is for fragmented elements only. If you want to contain 2+ elements in a single component, you wrap it like this.

```typescript
<>
  <A/>
  <B/>
</>
```

Otherwise, if the case where only 1 parent is needed, there's no need to use this.

```typescript
<A></A>
```